logstash-core/lib/logstash/pipeline.rb

```diff
@@ -72,8 +72,6 @@ def initialize(config_str, settings = {})
     @ready = Concurrent::AtomicBoolean.new(false)
     @running = Concurrent::AtomicBoolean.new(false)
     @flushing = Concurrent::AtomicReference.new(false)
-
-    start_flusher
   end # def initialize
 
   def ready?
```

Comment:
I think that can wait for a future refactor, I'd like to keep this scope to the bug at hand.